src/test/java/FooterTest.java

```diff
@@ -33,54 +31,61 @@ public void setUp() {
                 HOMEPAGE_URL + "sales/guest/form/");
-        validateFooterCategoriesNamesAndLinks(footerCategoriesList, expectedNamesList, expectedLinkList);
+        validateFooterCategoriesAndFooterNamesAndLinks(footerCategoriesList, expectedNamesList, expectedLinkList);
     }
 
     @Test
-    public void footer_SubscribeToNewsletterField_HasTextAndButtonCorrectlyDisplayed() {
+    public void footer_SubscribeToNewsletterForm_HasTextAndButtonCorrectlyDisplayed() {
         //then
         SoftAssertions softAssertions = new SoftAssertions();
         softAssertions.assertThat(footer.isSubscribeFormFieldDisplayed()).isTrue();
         softAssertions.assertThat(footer.isSubscribeButtonEnabled()).isTrue();
         softAssertions.assertThat(footer.getSubscribeButtonText()).isEqualTo("Subscribe");
         softAssertions.assertThat(footer.getEnterYourEmailPlaceHolderText()).isEqualTo("Enter your email address");
-        softAssertions.assertThat(footer.getSubscribeButtonFontColorAsHex()).isEqualTo(WHITE_COLOR);
-        softAssertions.assertThat(footer.getSubscribeButtonBackgroundColorAsHex()).isEqualTo(BLUE_COLOR);
+        softAssertions.assertThat(footer.getSubscribeButtonFontColorAsHex()).isEqualTo("#ffffff");
+        softAssertions.assertThat(footer.getSubscribeButtonBackgroundColorAsHex()).isEqualTo("#1979c3");
         softAssertions.assertAll();
     }
 
     @Test
     public void footer_FooterWebsiteInformation_HasTextCorrectlyDisplayed() {
         Assertions.assertThat(footer.getFooterWebsiteInformationText()).isEqualTo
-                ("Use this site to website to practice selenium.\n" +
-                        "This is a demo website for automation testing.\n" +
-                        "People also use it as a selenium practice site.\n" +
-                        "Searches,\n" +
-                        "shopping cart automation selenium");
+                ("""
+                        Use this site to website to practice selenium.
+                        This is a demo website for automation testing.
+                        People also use it as a selenium practice site.
+                        Searches,
+                        shopping cart automation selenium""");
+    }
+
+    @Test
+    public void footer_FooterCopyrightSection_HasTextCorrectlyDisplayed() {
+        Assertions.assertThat(footer.getFooterCopyrightInformationText()).isEqualTo
+                ("Copyright © 2013-present Magento, Inc. All rights reserved.");
+        Assertions.assertThat(footer.isCopyrightInformationDisplayed()).isTrue();
     }
 
-    public void validateFooterCategoriesNamesAndLinks(List<WebElement> footerCategoriesList, List<String> expectedNamesList,
+    public void validateFooterCategoriesAndFooterNamesAndLinks(List<WebElement> footerCategoriesList, List<String> expectedNamesList,
```

Comment:
List<String> expectedNamesList -> it was supposed to be changed to  List<String> expectedFooterCategoriesNamesList, not the method name.